validator/validator-main.protoascii

```diff
@@ -160,13 +160,8 @@ tags: {
     blacklisted_value_regex: "(^|\\s)("  # Values are space separated.
         "canonical|"  # Handled separately below, has specific requirements.
         "components|"
-        "dns-prefetch|"  # Handled separately below, has specific requirements.
```

Comment:
@Gregable @honeybadgerdontcare please check this out and the sections below (166 and followign and 205 and following) and make sure the change is what we want.